client/executor/common/src/wasm_runtime.rs

```diff
@@ -19,6 +19,38 @@
 use crate::error::Error;
 use sp_wasm_interface::Value;
 
+/// Call site.
+///
+/// Contains variants on how to resolve wasm function that will be invoked.
+pub enum CallSite<'a> {
```

Comment:
I am not sure if that's an appropriate name. "Call site" means a location in code from where some given function called. 

Should we just call it "an entrypoint"?